nevergrad/functions/unit_commitment/__init__.py

```diff
@@ -0,0 +1 @@
+from .core import UnitCommitmentProblem1 as UnitCommitmentProblem1
```

Comment:
for pep8 compliance, please rename the directory from `unit_commitment` to `unitcommitment`